docker/makefile

```diff
@@ -0,0 +1,12 @@
+# Makefile
```

Comment:
There are some syntax errors in the file I think?

Reply:
Yeah, my vscode made them used normal spaces for tabspaces, I have changed it in settings and is working fine now.